tic_tac_toe/game.py

```diff
@@ -87,18 +129,61 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
+    board = game['board']
+    if game['winner'] or _board_is_full(board):
+        raise InvalidMovement("Game is over.")
+        
+    elif not _position_is_valid(position):
+        raise InvalidMovement('Position out of range.')
+    
+    elif not _position_is_empty_in_board(position, board):
+        raise InvalidMovement('Position already taken.')
+    
+    elif get_next_turn(game) != player:
+        raise InvalidMovement('"{}" moves next'.format(game['next_turn']))
+    
+    
+    board[position[0]][position[1]] = player
+    
+    winner = _check_winning_combinations(board, player)
+    
+    if winner:
+        game['winner'] = winner
+        game['next_turn'] = None
```

Comment:
It is not necessary to set `next_turn` to None as you never check if it's None, you only check if the player trying to make a move is the same as the player in `next_turn`. You also never reach that comparison(if `next_turn` is `player`) if you've set `next_turn` to None as in that case the game is either won or tied, which is covered by your first if statement.